### PR TITLE
Geonode 2.6 installation with postgres 9.5 

### PR DESCRIPTION
**Pull request by hascar**
Geonode 2.6 installation with postgres 9.5 https://ci.openquake.org/job/zdevel_oq-platform2/36/console

### CODE REVIEW

File: verifier-guest.sh [hunk 1]
```diff
@@ -47,6 +47,25 @@ source ~/env/bin/activate
...
 cd ~
 
+#install and configuration postgres
+sudo apt-get install -y postgresql-9.5-postgis-2.2 postgresql-9.5-postgis-scripts
+sudo -u postgres createdb geonode_dev
+sudo -u postgres createdb geonode_dev-imports
+cat << EOF | sudo -u postgres psql
+CREATE USER "geonode_dev" WITH PASSWORD 'geonode_dev';
+GRANT ALL PRIVILEGES ON DATABASE "geonode_dev" to geonode_dev;
+GRANT ALL PRIVILEGES ON DATABASE "geonode_dev-imports" to geonode_dev;
```
Comment (nastasi-oq) [inline verifier-guest.sh]: Please use variables for 'database name', 'database user' and so on.

File: verifier-guest.sh [hunk 2]
```diff
@@ -87,6 +106,8 @@ if [ "$NO_EXEC_TEST" != "notest" ] ; then
     exec_test
 fi
 
+#sleep 40000
+
```
Comment (nastasi-oq) [inline verifier-guest.sh]: Please remove devel commented sleep as final commit

File: verifier-guest.sh (hunk 1 above)
Comment (hascar) [inline verifier-guest.sh]: Corrected here https://github.com/gem/oq-platform2/pull/6/commits/177985cc1d5031f16d9954df11f904d80cd1c1d2

File: verifier-guest.sh (hunk 2 above)
Comment (hascar) [inline verifier-guest.sh]: Corrected here https://github.com/gem/oq-platform2/pull/6/commits/f01aec3e97e02bd4e41254864bddb0fa8d5a5177

**Pull request merged**